src/store/modules/ui.store.js

```diff
@@ -134,6 +141,19 @@ export default {
         isTraditionalDesktopSize(state, getters) {
             return getters.isDesktopMode && state.width >= BREAKPOINT_TABLET
         },
+        /**
+         * Flag to display a warning ribbon ('TEST') at the top/bottom right corner
+         */
+        hasWarningRibbon(state) {
+            return WARNING_RIBBON_HOSTNAMES.some(hostname=>state.hostname.includes(hostname))
+        },
+        /**
+        * Flag that tells if users should be warned that it is a development site. Also used to hide
+        * development specific features in production (like the app version)
+        */
+        devSiteWarning(state) {
```

Comment:
Because this is a boolean and also to keep similar nomenclature as for the ribbon, I would name this variable `hasDevSiteWarning`

Reply:
OK, makes sense.